src/jam.c

```diff
@@ -0,0 +1,144 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <assert.h>
+
+#define JAMMED_FILE "jammed.js"
+
+#define NUMBER_OF_JS 9
+
+
+#define file0 "core.js"
+#define file1 "line.js"
+#define file2 "inheritance_line.js"
+#define file3 "diagram.js"
+#define file4 "property_diagram.js"
+#define file5 "variable_property.js"
+#define file6 "method_property.js"
+#define file7 "test.js"
```

Comment:
test.js should not be packed, it should be loaded in the html, as it instantiate the blocks for that specific html.